tests/spec_test.py

```diff
@@ -1,4 +1,4 @@
-"""Tests for core spec objects."""
+'''Tests for core spec objects.'''
```

Comment:
We explicitly [ignore](https://github.com/girder/girder_worker/blob/3df3b2edf6f0718b3c79a81770afe824abf7aea9/tests/flake8.cfg#L56) the flake8-docstrings `D300 - Use """triple double quotes"""` test, but is there a reason to change this file's docstrings to use single quotes?